README.md

```diff
@@ -64,30 +100,34 @@ VITE_MILLICAST_STREAM_ID=<your stream id>
 The viewer link is configured in the `.env` file in your `apps/publisher` directory, or in your bash/zsh/systen environment variables. To set the URL, insert/update the following variable:
 
 ```bash
-VITE_MILLICAST_VIEWER_BASE_URL=<Your URL goes here>
+VITE_RTS_VIEWER_BASE_URL=<Your URL goes here>
 ```
 
-> If you are using a particular port number, please add this to the URL.
+> If you are using a particular port number, please add this to the URL. The best way to do this is to first [run the viewer app](#running-the-viewer-app), grab the URL from your browser and add it to your `.env` file as such.
 
 ```bash
-VITE_MILLICAST_VIEWER_BASE_URL=http://localhost:5174/
+VITE_RTS_VIEWER_BASE_URL=http://localhost:5174/
 ```
 
-#### Run the publisher app in dev mode
+#### Running the publisher app in dev mode
 
 ```bash
 yarn nx serve publisher
 ```
 
-#### Build the publisher app and preview it
+**Note**: Remember to grant the publisher app all the necessary camera and microphone permissions.
+
+![permissions](docs/img/permissions.png)
```

Comment:
@dubeyShivank 
Please use the latest screenshot of the app, this screenshot has milicast title name.

Reply:
Good observation! Thank you @vishalkharge